src/Model/Button/AccountLink.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace BokaiTid\FBMessenger\Model\Button;
+
+use Tgallice\FBMessenger\Model\Button;
+
+class AccountLink extends Button
+{
+    /**
+     * @var string $url
+     */
+    private $url;
+
+    /**
+     * @param string $url
+     */
+    public function __construct($url)
+    {
+        parent::__construct(Button::TYPE_ACCOUNT_LINK);
+
+        $this->url = $url;
+    }
+
+    /**
+     * @return string
+     */
+    public function getUrl()
+    {
+        return $this->url;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function jsonSerialize()
+    {
+        $json = parent::jsonSerialize();
+        $json['url'] = $this->url;
+
+        return $json;
+    }
+}
```

Comment:
and you will need to force the update, because your local branch dont match your remote branch. So:
```
// -f flag for force, which will override your remote branch
// this can be dangerous so take care when you force an update
$ git push origin branch_name -f
```